highlight/events-explainer.md

```diff
@@ -95,13 +96,67 @@ Whenever an element receives pointer input, if a range within that element is al
 
 Elements and Highlights should receive the same pointer events. This means that a Highlight callback would be able to prevent the event from reaching an element by calling stopPropogation, as well as canceling the event via preventDefault.
 
-As an example, consider a scenario where there are two consecutive paragraphs, p1 and p2, with one highlight, h1, on the last line of p1 and another highlight, h2, on the first line of p2. Let's say the user performs a mouse down on the last line of p1, moves the mouse to the first line of p2, and then performs a mouse up. The sequence of events fired would be:
+As an example, consider a scenario where there are two consecutive paragraphs, p1 and p2, with one highlight, h1, on the last line of p1 and another highlight, h2, on the first line of p2.
+
+```html
+<html>
+<head>
+    <style>
+        :root::highlight(foo) {
+            background-color: red;
+        }
+        :root::highlight(bar) {
+            background-color: blue;
+        }
+    </style>
+</head>
+<body>
+    <p id="p1">foo</p>
+    <p id="p2">bar</p>
+    <script type="module">
+        const h1 = new Highlight()
+        CSS.highlights.set("foo", h1)
+        const h2 = new Highlight()
+        CSS.highlights.set("bar", h2)
+
+        const p1 = document.getElementById("p1")
+        const r1 = new Range()
+        r1.setStart(p1, 0)
+        r1.setEnd(p1, 3)
+        h1.add(r1)
+
+        const p2 = document.getElementById("p2")
+        const r2 = new Range()
+        r2.setStart(p2, 0)
+        r2.setEnd(p2, 3)
+        h2.add(r2)
+    </script>
+</body>
+</html>
+```
+
+Let's say the user performs a mouse down on the last line of p1, moves the mouse to the first line of p2, and then performs a mouse up. The sequence of events fired would be:
 
 ```
-h1 pointerdown, p1 pointerdown
-h1 pointermove(s), p1 pointermove(s)
-h2 pointermove(s), p2 pointermove(s) - assuming p1 does not take pointer capture
-h2 pointerup, p2 pointerup
+PointerDown p1
+    Capture phase p1 pointerdown - target:p1 currentTarget:p1 
+    Capture\Bubbling phase h1 pointerdown - target:p1 currentTarget:h1 range:r1
+    Bubbling phase p1 pointerdown - target:p1 currentTarget:p1
+
+PointerMove(s) p1
+    Capture phase p1 pointermove - target:p1 currentTarget:p1 
+    Capture\Bubbling phase h1 pointermove - target:p1 currentTarget:h1 range:r1

```

Comment:
I think this is usually called the "Target phase".

Reply:
That's true, but it seems odd to call it "Target phase" when the 'Highlight' is not the 'target'. Would this not cause confusion?